treescript/versionmanip.py

```diff
@@ -54,43 +55,46 @@ def _get_version(file):
 
     """
     bump_info = get_version_bump_info(context.task)
-    next_version = bump_info['next_version']
-    old_next_version = None
     files = bump_info['files']
     changed = False
+
+    VersionClass = _find_what_version_parser_to_use(files)
```

Comment:
I'd rather we check the parser based on file to bump - specifically milestone.txt is a gecko version and I can envision it being bumped with different products in the future, should we get there... thoughts?

Reply:
Sure, I can use `GeckoVersion` for `milestones.txt`! Good idea!

Reply:
Done!